ext/snmp/snmp.c

```diff
@@ -1677,19 +1677,30 @@ zval *php_snmp_write_property(zend_object *object, zend_string *name, zval *valu
 	obj = php_snmp_fetch_object(object);
```

Comment:
Should I also add a `get_property_ptr_ptr` handler, right?

Reply:
That's right. It should return NULL for "special" properties.